src/sys/ptrace/linux.rs

```diff
@@ -58,7 +58,8 @@ libc_enum! {
                                                target_arch = "mips64",
                                                target_arch = "mips64r6",
                                                target_arch = "x86_64",
-                                               target_pointer_width = "32"))))]
```

Comment:
Nit: this change looks good, but the readability would be better if we move `target_env = "ohos"` under item `target_env = "musl"`

Reply:
This also applies to the remaining changes in this file